packages/google_maps_flutter/google_maps_flutter_ios/example/ios14/ios/RunnerTests/GoogleMapsTests.m

```diff
@@ -62,4 +66,20 @@ - (void)testMapsServiceSync {
   XCTAssertEqual(factory1.sharedMapServices, factory2.sharedMapServices);
 }
 
+- (void)testWideGamutImagesProperlyHandled {
+  FLTTileProviderController *controller = [[FLTTileProviderController alloc] init];
+
+  UIImage *wideGamutImage = [UIImage imageWithContentsOfFile:@"assets/widegamut.png"];
+
+  UIImage *downsampledImage = [controller handleResultTile:wideGamutImage];
+
+  CGImageRef imageRef = downsampledImage.CGImage;
+  CGBitmapInfo bitmapInfo = CGImageGetBitmapInfo(imageRef);
+  bool isFloat = (bitmapInfo && kCGBitmapFloatComponents);
+  size_t bitsPerComponent = CGImageGetBitsPerComponent(imageRef);
+
+  XCTAssert(isFloat == false);
+  XCTAssert(bitsPerComponent != 16);
```

Comment:
what's the value actually gonna be?

Reply:
non-wide gamut imgs are 8 bit float format